deployment/nginx/cchq_local_nginx.conf

```diff
@@ -28,6 +34,14 @@ server {
         try_files $uri $uri/ @proxy_to_app;
     }
 
+    # uncomment for formplayer
+    # location ~ ^/formplayer/(?<domain>[\w-]+)/(?<username>[\w-\.@]+)/(?<route>[^\/]*) {
+    #     set $request_hash $domain$username;
+    #     proxy_pass http://formplayer/$route;
+    #     proxy_read_timeout 900s;
+    #     proxy_next_upstream_tries 1;
+    # }
```

Comment:
this is essentially what the nginx config would look like. the location capture pulls out the `domain`, `username` and the rest of the route (`route`). we take the domain and username and set it to a variable we'd like to hash. then we do a proxy pass with remainder of the route to get the correct result